common-test/src/main/java/io/strimzi/test/CmData.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation for test classes or methods run via {@code @RunWith(StrimziRunner.class)}
+ * which causes that runner to update config map before tests
+ */
+@Target({ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Repeatable(CmData.Container.class)
```

Comment:
I don't think this annotation should be `@Repeatable` (so we can lose the Container inner class as well). Instead there should be a `CmData[] config()` member in `@KafkaCluster`. That way you would specify a single `@KafkaCluster` with all the configuration defined within it. That seems more logical to me.